### PR TITLE
Adding vanilla-lazyload to the scripts at the end of the page

### PR DESCRIPTION
**Pull request by verlok**
Proposing the addition of vanilla-lazyload

Changes proposed in this pull request:

- Added vanilla-lazyload to the list of scripts at the end of the article

### CODE REVIEW

**Comment by verlok** [comment]
@googlebot I signed it!

**Comment by verlok** [comment]
Sorry for the delay, I had signed the CLA but I thought the sign check was automatic.
Would you be able to review it now?

**Review by samthor — COMMENTED** [review]
In general I think this is fine since we're linking to a variety of libraries here. @kaycebasques  ?

**Comment by samthor** [comment]
🤖  active

**Comment by verlok** [comment]
Hey @kaycebasques, what do you think? Can we merge this in?

File: src/site/content/en/fast/lazy-loading-images/index.md [hunk 1]
```diff
@@ -274,7 +274,11 @@ into `src` and/or `srcset` attributes, respectively. It uses Intersection
 Observer (which you can polyfill), and can be extended with [a number of
 plugins](https://github.com/aFarkas/lazysizes#available-plugins-in-this-repo) to
 do things like lazy-load video. [Find out more about using lazysizes](/use-lazysizes-to-lazyload-images/).
-- [lozad.js](https://github.com/ApoorvSaxena/lozad.js) is a super lightweight
+- [vanilla-lazyload](https://github.com/verlok/vanilla-lazyload) is a super 
+lightweight option for lazy loading images, background images, videos, iframes 
+and scripts. It leverages IntersectionObserver, supports responsive images and 
+enables native lazy loading.
```
Comment (mihajlija) [inline src/site/content/en/fast/lazy-loading-images/index.md:280]:
```suggestion
- [vanilla-lazyload](https://github.com/verlok/vanilla-lazyload) is a 
lightweight option for lazy loading images, background images, videos, iframes, 
and scripts. It leverages IntersectionObserver, supports responsive images, and 
enables native lazy loading.
```

**Review by mihajlija — APPROVED** [review]

File: src/site/content/en/fast/lazy-loading-images/index.md [hunk 2]
```diff
@@ -274,7 +274,11 @@ into `src` and/or `srcset` attributes, respectively. It uses Intersection
 Observer (which you can polyfill), and can be extended with [a number of
 plugins](https://github.com/aFarkas/lazysizes#available-plugins-in-this-repo) to
 do things like lazy-load video. [Find out more about using lazysizes](/use-lazysizes-to-lazyload-images/).
-- [lozad.js](https://github.com/ApoorvSaxena/lozad.js) is a super lightweight
+- [vanilla-lazyload](https://github.com/verlok/vanilla-lazyload) is a 
+lightweight option for lazy loading images, background images, videos, iframes,
```
Comment (mihajlija) [inline src/site/content/en/fast/lazy-loading-images/index.md:278]:
```suggestion
lightweight option for lazy-loading images, background images, videos, iframes, 
```

File: src/site/content/en/fast/lazy-loading-images/index.md [hunk 3]
```diff
@@ -274,7 +274,11 @@ into `src` and/or `srcset` attributes, respectively. It uses Intersection
 Observer (which you can polyfill), and can be extended with [a number of
 plugins](https://github.com/aFarkas/lazysizes#available-plugins-in-this-repo) to
 do things like lazy-load video. [Find out more about using lazysizes](/use-lazysizes-to-lazyload-images/).
-- [lozad.js](https://github.com/ApoorvSaxena/lozad.js) is a super lightweight
+- [vanilla-lazyload](https://github.com/verlok/vanilla-lazyload) is a 
+lightweight option for lazy-loading images, background images, videos, iframes, 
+and scripts. It leverages Intersection Observer, supports responsive images, and 
+enables native lazy loading.
```
Comment (kaycebasques) [inline src/site/content/en/fast/lazy-loading-images/index.md:280]:
```suggestion
- [vanilla-lazyload](https://github.com/verlok/vanilla-lazyload) is a 
  lightweight option for lazy-loading images, background images, videos, iframes, 
  and scripts. It leverages Intersection Observer, supports responsive images, and 
  enables native lazy loading.
```

**Review by kaycebasques — APPROVED** [review]
LGTM. Thanks for the update (and the library) @verlok.

**Comment by verlok** [comment]
You’re welcome!

What happens next? Do we need the approval of @jpmedley too, or is this ready to be merged?

**Pull request merged**